src/Console/Command/PackageReleaseCommand.php

```diff
@@ -0,0 +1,380 @@
+
+            // the full command description shown when running the command with
+            // the "--help" option
+            ->setHelp(
+                'This tool is used to release new versions of composer '
+                . 'packages. It uses Semver 2.0 to automatically pick the '
+                . 'next version number and tag the release on GitHub.'
+            )
+            ->setDefinition(
+                new InputDefinition(array(
+                    new InputOption(
+                        'branch',
+                        'b',
+                        InputOption::VALUE_REQUIRED,
+                        'Remote branch to use for release.',
+                        'master'
+                    ),
+                    new InputOption(
+                        'message',
+                        'm',
+                        InputOption::VALUE_REQUIRED,
+                        'Message to use for the release tag.'
+                    ),
+                    new InputOption(
+                        'type',
+                        't',
+                        InputOption::VALUE_REQUIRED,
+                        'Release type. Must be one of "major", "minor", or '
+                        . '"patch". Semver 2.0 (https://semver.org/) is used '
+                        . 'to pick the next release number.',
+                        'minor'
+                    ),
+                ))
+            );
+        ;
+    }
+
+    protected function execute(
+        InputInterface $input,
+        OutputInterface $output
+    ): int {
+        $this->validateInputOptions($input, $output);
+
+        // Use custom OutputFormatter that works with mbstring function
+        // overloading enabled.
+        $formatter = $output->getFormatter();
+        $output->setFormatter(
+            new PackageReleaseOutputFormatter($formatter->isDecorated())
+        );
+
+        $this->style->execute($input, $output);
+
+        if (!$this->manager->isInGitRepo()) {
+            $output->writeln([
+                'This tool must be run from a git repository.',
+                '',
+            ]);
+            return 1;
+        }
+
+        if (!$this->manager->isComposerPackage()) {
+            $output->writeln([
+                'Could not find <variable>composer.json</variable>. Make '
+                . 'sure you are in the project '
+                . 'root and the project is a composer package.',
+                '',
+            ]);
+            return 1;
+        }
+
+        $repo_name = $this->manager->getRepoName();
+        if ($repo_name === null) {
+            $output->writeln([
+                'Could not find git repository name. Git repository '
+                . 'must have a remote named <variable>origin</variable>.',
+                '',
+            ]);
+            return 1;
+        }
+
+        $remote_url = sprintf(
+            'git@github.com:silverorange/%s.git',
```

Comment:
I think this will also fail for our repos on other orgs (like `uas-data-store`).

Reply:
Ups and we don't use this tool for that. Do we/will we have any PHP packages on other orgs? I was thinking at some point we wanted to move the hippo-specific code over into HippoEducation (including things like Kos) but that's all I'm aware of.

Reply:
We don't currently have PHP packages on other GitHub orgs. This matches the current behaviour which needs fixing. Opened issue https://github.com/silverorange/package_release/issues/13